ts/Data/Formula/Functions/COUNT.ts

```diff
@@ -0,0 +1,110 @@
+/* *
+ *
+ *  (c) 2009-2023 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Sophie Bremer
+ *
+ * */
+
+
+'use strict';
+
+/* eslint-disable new-cap */
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+
+import type {
+    Arguments,
+    Value
+} from '../FormulaTypes';
+import type DataTable from '../../DataTable';
+
+
+import FormulaProcessor from '../FormulaProcessor.js';
+
+
+/* *
+ *
+ *  Functions
+ *
+ * */
+
+
+/**
+ * Processor for the `COUNT(...values)` implementation. Returns the count of
+ * given values that are numbers.
+ *
+ * @private
+ * @function Formula.processorFunctions.COUNT
+ *
+ * @param {Highcharts.FormulaArguments} args
+ * Arguments to process.
+ *
+ * @param {Highcharts.DataTable} [table]
+ * Table to use for references and ranges.
+ *
+ * @return {number}
+ * Result value of the process.
+ */
+function COUNT(
+    args: Arguments,
+    table?: DataTable
+): number {
+    const values = FormulaProcessor.getArgumentsValues(args, table);
+
+    let count = 0;
+
+    for (
+        let i = 0,
+            iEnd = values.length,
+            value: (Value|Array<Value>);
+        i < iEnd;
+        ++i
+    ) {
+        value = values[i];
+
+        switch (typeof value) {
```

Comment:
Do we need switch here? We have two cases like in IF / ELSEIF

Reply:
You safe one typeof.